docs/tutorial/persistable-classes.txt

```diff
@@ -0,0 +1,154 @@
+===================
+Persistable classes
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+The driver's :php:`persistence specification <mongodb.persistence>` outlines how
```

Comment:
Currently, this page has only a single "Type Maps" section. I'd suggest adding a heading here:

```suggestion
Persistable Classes
-------------------

The driver's :php:`persistence specification <mongodb.persistence>` outlines how
```

Additionally, I propose moving "Type Maps" _above_ this section, since this section refers back to the previous (e.g. "without requiring the use of the ``typeMap`` option"). Users should read and understand how type maps work before jumping into Persistable classes.

Reply:
I did both: added a header for persistable classes and moved the type map documentation up before persistable classes.